airbyte-cdk/python/airbyte_cdk/sources/file_based/remote_file.py

```diff
@@ -3,11 +3,17 @@
 #
 
 from datetime import datetime
+from enum import Enum
 from typing import Optional
 
 from pydantic import BaseModel
 
 
+class FileReadMode(Enum):
```

Comment:
This will need to be released before the airbyte-integrations as it will not be available for the source. Consequence: if someone needs to release source-s3, the tests will fail